rtl_wmbus.c

```diff
@@ -780,9 +819,7 @@ int main(int argc, char *argv[])
 
             if (opts_s1_t1_c1_simultaneously)
             {
-                shift_freq_250(&i_t1_c1_unfilt, &q_t1_c1_unfilt, fs_kHz);
-
-                shift_freq_minus400(&i_s1_unfilt, &q_s1_unfilt, fs_kHz);
+            shift_freq_plus_minus325(&i_t1_c1_unfilt, &q_t1_c1_unfilt,&i_s1_unfilt, &q_s1_unfilt, fs_kHz);
```

Comment:
As I explained above, two frequency translations were put into one function. It's an open question if this is a speed up, but surely less readable.

Reply:
Verified now with profiler gprof, speeds up but it is not the main bottleneck